pkg/kube/inject/testdata/webhook/daemonset.yaml.injected

```diff
@@ -18,6 +18,7 @@ spec:
       labels:
         app: hello
         security.istio.io/tlsMode: istio
+        service.istio.io/canonical-name: hello
```

Comment:
Do we need a label or would annotation work? Are we querying by this?

Reply:
@smawson I believe that we want this to be a label and not an annotation. we are not querying by this today, but may in the future.

Reply:
Yes and we want to support users selecting workloads based on this, and we don't allow selecting based on annotations. So yeah, label is right.